make distcheck fails

make distcheck is failing for me on Mac OS and Linux:

```
/usr/bin/make  dist-gzip am__post_remove_distdir='@:'
make[1]: *** No rule to make target `win/rtptools.suo', needed by `distdir'.  Stop.
make: *** [dist] Error 2
```

But given #50, maybe this is a "Won't Fix".

Sorry, still not working.

Output from Debian Linux 9.4:
```
...
make[1]: Entering directory '/home/njh/Projects/rtptools/rtptools-1.22/_build/sub'
 ( cd '/tmp/am-dc-22948//home/njh/Projects/rtptools/rtptools-1.22/_inst/bin' && rm -f rtpdump rtpplay rtpsend rtptrans )
 ( cd '/tmp/am-dc-22948//home/njh/Projects/rtptools/rtptools-1.22/_inst/bin' && rm -f multidump multiplay )
 ( cd '/tmp/am-dc-22948//home/njh/Projects/rtptools/rtptools-1.22/_inst/share/man/man1' && rm -f rtpdump.1 rtpplay.1 rtpsend.1 rtptrans.1 multidump.1 multiplay.1 )
make[1]: Leaving directory '/home/njh/Projects/rtptools/rtptools-1.22/_build/sub'
make[1]: Entering directory '/home/njh/Projects/rtptools/rtptools-1.22/_build/sub'
make[1]: Leaving directory '/home/njh/Projects/rtptools/rtptools-1.22/_build/sub'
make[1]: Entering directory '/home/njh/Projects/rtptools/rtptools-1.22/_build/sub'
make  dist-gzip am__post_remove_distdir='@:'
make[2]: Entering directory '/home/njh/Projects/rtptools/rtptools-1.22/_build/sub'
make[2]: *** No rule to make target 'win/*.c', needed by 'distdir'.  Stop.
make[2]: Leaving directory '/home/njh/Projects/rtptools/rtptools-1.22/_build/sub'
Makefile:712: recipe for target 'dist' failed
make[1]: *** [dist] Error 2
make[1]: Leaving directory '/home/njh/Projects/rtptools/rtptools-1.22/_build/sub'
Makefile:719: recipe for target 'distcheck' failed
make: *** [distcheck] Error 1
```

The error message is causing the failure.
"make[2]: *** No rule to make target 'win/*.c', needed by 'distdir'.  Stop."
But I clearly see having 'win/*.c' in the github repo and the file downloaded from github.
From both.
"Clone or download" button here
https://github.com/columbia-irt/rtptools
and 
git cloning and the command line.

Do you mind teaching me how you downloading the files?

Confirmed no errors in the new build system:

```
~/Projects/rtptools(master) $ make distclean
rm -f rtptools-1.23.tar.gz rtpdump rtpplay rtpsend rtptrans hpt.o host2ip.o                     rd.o rtpdump.o hpt.o host2ip.o notify.o multimer.o rd.o rtpplay.o hpt.o host2ip.o notify.o multimer.o      rtpsend.o hpt.o host2ip.o notify.o multimer.o      rtptrans.o compat_err.o compat_strtonum.o multidump.1.html multiplay.1.html rtpdump.1.html rtpplay.1.html rtpsend.1.html rtptrans.1.html
rm -rf rtptools-1.23 win/include
rm -rf *.dSYM *.core *~ .*~
rm -f Makefile.local config.h config.h.old config.log config.log.old
~/Projects/rtptools(master) $ ./configure 
config.log: writing...
configure.local: no (fully automatic configuration)
gethostbyname-lnsl: no
msgcontrol: yes
socket-lsocket: no
config.h: written
Makefile.local: written
~/Projects/rtptools(master) $ make distcheck
mkdir -p win/include/{arpa,netinet,sys}
touch win/include/arpa/inet.h
mkdir -p win/include/{arpa,netinet,sys}
touch win/include/netdb.h
mkdir -p win/include/{arpa,netinet,sys}
touch win/include/netinet/in.h
mkdir -p win/include/{arpa,netinet,sys}
touch win/include/sys/select.h
mkdir -p win/include/{arpa,netinet,sys}
touch win/include/sys/socket.h
mkdir -p win/include/{arpa,netinet,sys}
touch win/include/sys/time.h
mkdir -p win/include/{arpa,netinet,sys}
touch win/include/sys/uio.h
mkdir -p win/include/{arpa,netinet,sys}
touch win/include/unistd.h
rm -rf .dist
mkdir -p .dist/rtptools-1.23/
install -m 0644 LICENSE Makefile Makefile.depend bark.rtp configure configure.local.example ChangeLog.html multidump.1 multiplay.1 rtpdump.1 rtpplay.1 rtpsend.1 rtptrans.1 multidump multiplay ansi.h host2ip.c hpt.c hsearch.c hsearch.h multimer.c multimer.h notify.c notify.h rd.c rtp.h rtpdump.c rtpdump.h rtpplay.c rtpsend.c rtptrans.c sysdep.h vat.h have-err.c have-gethostbyname.c have-strtonum.c have-msgcontrol.c .dist/rtptools-1.23/
cp -r win .dist/rtptools-1.23/
( cd .dist/rtptools-1.23 && chmod 755 configure multidump multiplay )
( cd .dist && tar czf ../rtptools-1.23.tar.gz rtptools-1.23 )
rm -rf .dist/
rm -rf rtptools-1.23 && tar xzf rtptools-1.23.tar.gz
( cd rtptools-1.23 && ./configure && make all )
config.log: writing...
configure.local: no (fully automatic configuration)
gethostbyname-lnsl: no
msgcontrol: yes
socket-lsocket: no
config.h: written
Makefile.local: written
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c hpt.c
hpt.c:44:5: warning: no previous prototype for function 'hpt'
      [-Wmissing-prototypes]
int hpt(char *h, struct sockaddr *sa, unsigned char *ttl)
    ^
1 warning generated.
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c host2ip.c
host2ip.c:43:16: warning: no previous prototype for function 'host2ip'
      [-Wmissing-prototypes]
struct in_addr host2ip(char *host)
               ^
host2ip.c:52:42: warning: comparison of integers of different signs: 'in_addr_t'
      (aka 'unsigned int') and 'int' [-Wsign-compare]
  else if ((in.s_addr = inet_addr(host)) != -1) {
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~  ^  ~~
2 warnings generated.
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c rd.c
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c rtpdump.c
rtpdump.c:114:11: warning: passing 'const char [1]' to parameter of type
      'char *' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    usage("");
          ^~
rtpdump.c:66:25: note: passing argument to parameter 'argv0' here
static void usage(char *argv0)
                        ^
rtpdump.c:119:11: warning: passing 'const char [1]' to parameter of type
      'char *' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    usage("");
          ^~
rtpdump.c:66:25: note: passing argument to parameter 'argv0' here
static void usage(char *argv0)
                        ^
rtpdump.c:117:28: warning: comparison of integers of different signs:
      'in_addr_t' (aka 'unsigned int') and 'int' [-Wsign-compare]
  if (sin->sin_addr.s_addr == -1) {
      ~~~~~~~~~~~~~~~~~~~~ ^  ~~
rtpdump.c:219:12: warning: returning 'const char *' from a function with result
      type 'char *' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    return r->version == RTP_VERSION ? "RTP" : "VATD";
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rtpdump.c:223:12: warning: returning 'const char *' from a function with result
      type 'char *' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    return r->common.version == RTP_VERSION ? "RTCP" : "VATC";
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rtpdump.c:289:43: warning: format specifies type 'unsigned long' but the
      argument has type 'uint32_t' (aka 'unsigned int') [-Wformat]
      fprintf(out, "csrc[%d]=0x%0lx ", i, r->csrc[i]);
                               ~~~~       ^~~~~~~~~~
                               %0x
rtpdump.c:322:46: warning: format specifies type 'long' but the argument has
      type '__darwin_suseconds_t' (aka 'int') [-Wformat]
      (v->flags ? -now.tv_sec : now.tv_sec), now.tv_usec,
                                             ^~~~~~~~~~~
rtpdump.c:327:42: warning: format specifies type 'long' but the argument has
      type '__darwin_suseconds_t' (aka 'int') [-Wformat]
      (r->m ? -now.tv_sec : now.tv_sec), now.tv_usec,
                                         ^~~~~~~~~~~
rtpdump.c:345:24: warning: initializing 'char *' with an expression of type
      'const char [4]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_END,    "end"}, 
                       ^~~~~
rtpdump.c:346:24: warning: initializing 'char *' with an expression of type
      'const char [6]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_CNAME,  "CNAME"},
                       ^~~~~~~
rtpdump.c:347:24: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_NAME,   "NAME"},
                       ^~~~~~
rtpdump.c:348:24: warning: initializing 'char *' with an expression of type
      'const char [6]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_EMAIL,  "EMAIL"},
                       ^~~~~~~
rtpdump.c:349:24: warning: initializing 'char *' with an expression of type
      'const char [6]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_PHONE,  "PHONE"},
                       ^~~~~~~
rtpdump.c:350:24: warning: initializing 'char *' with an expression of type
      'const char [4]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_LOC,    "LOC"},
                       ^~~~~
rtpdump.c:351:24: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_TOOL,   "TOOL"},
                       ^~~~~~
rtpdump.c:352:24: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_NOTE,   "NOTE"},
                       ^~~~~~
rtpdump.c:353:24: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {RTCP_SDES_PRIV,   "PRIV"},
                       ^~~~~~
rtpdump.c:354:24: warning: initializing 'char *' with an expression of type
      'const char [7]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {11,               "SOURCE"},
                       ^~~~~~~~
rtpdump.c:339:6: warning: no previous prototype for function 'member_sdes'
      [-Wmissing-prototypes]
void member_sdes(FILE *out, member_t m, rtcp_sdes_type_t t, char *b, int len)
     ^
rtpdump.c:581:29: warning: format specifies type 'long' but the argument has
      type '__darwin_suseconds_t' (aka 'int') [-Wformat]
                now.tv_sec, now.tv_usec, parse_type(ctrl, packet->p.data),
                            ^~~~~~~~~~~
rtpdump.c:594:29: warning: format specifies type 'long' but the argument has
      type '__darwin_suseconds_t' (aka 'int') [-Wformat]
                now.tv_sec, now.tv_usec, parse_type(ctrl, packet->p.data),
                            ^~~~~~~~~~~
rtpdump.c:527:6: warning: no previous prototype for function 'packet_handler'
      [-Wmissing-prototypes]
void packet_handler(FILE *out, t_format format, int trunc,
     ^
rtpdump.c:613:6: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"dump",    F_dump},
     ^~~~~~
rtpdump.c:614:6: warning: initializing 'char *' with an expression of type
      'const char [7]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"header",  F_header},
     ^~~~~~~~
rtpdump.c:615:6: warning: initializing 'char *' with an expression of type
      'const char [4]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"hex",     F_hex},
     ^~~~~
rtpdump.c:616:6: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"rtcp",    F_rtcp},
     ^~~~~~
rtpdump.c:617:6: warning: initializing 'char *' with an expression of type
      'const char [6]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"short",   F_short},
     ^~~~~~~
rtpdump.c:618:6: warning: initializing 'char *' with an expression of type
      'const char [8]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"payload", F_payload},
     ^~~~~~~~~
rtpdump.c:619:6: warning: initializing 'char *' with an expression of type
      'const char [6]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"ascii",   F_ascii},
     ^~~~~~~
rtpdump.c:713:20: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
    pt_map[i].enc  = "????";
                   ^ ~~~~~~
rtpdump.c:720:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  0].enc = "PCMU"; pt_map[  0].rate =  8000; pt_map[  0].ch = 1;
                  ^ ~~~~~~
rtpdump.c:721:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  1].enc = "1016"; pt_map[  1].rate =  8000; pt_map[  1].ch = 1;
                  ^ ~~~~~~
rtpdump.c:722:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  2].enc = "G721"; pt_map[  2].rate =  8000; pt_map[  2].ch = 1;
                  ^ ~~~~~~
rtpdump.c:723:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  3].enc = "GSM "; pt_map[  3].rate =  8000; pt_map[  3].ch = 1;
                  ^ ~~~~~~
rtpdump.c:724:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  4].enc = "G723"; pt_map[  4].rate =  8000; pt_map[  4].ch = 1;
                  ^ ~~~~~~
rtpdump.c:725:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  5].enc = "DVI4"; pt_map[  5].rate =  8000; pt_map[  5].ch = 1;
                  ^ ~~~~~~
rtpdump.c:726:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  6].enc = "DVI4"; pt_map[  6].rate = 16000; pt_map[  6].ch = 1;
                  ^ ~~~~~~
rtpdump.c:727:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  7].enc = "LPC "; pt_map[  7].rate =  8000; pt_map[  7].ch = 1;
                  ^ ~~~~~~
rtpdump.c:728:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  8].enc = "PCMA"; pt_map[  8].rate =  8000; pt_map[  8].ch = 1;
                  ^ ~~~~~~
rtpdump.c:729:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[  9].enc = "G722"; pt_map[  9].rate =  8000; pt_map[  9].ch = 1;
                  ^ ~~~~~~
rtpdump.c:730:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 10].enc = "L16 "; pt_map[ 10].rate = 44100; pt_map[ 10].ch = 2;
                  ^ ~~~~~~
rtpdump.c:731:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 11].enc = "L16 "; pt_map[ 11].rate = 44100; pt_map[ 11].ch = 1;
                  ^ ~~~~~~
rtpdump.c:732:19: warning: assigning to 'char *' from 'const char [6]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 12].enc = "QCELP"; pt_map[ 12].rate = 8000; pt_map[ 12].ch = 1;
                  ^ ~~~~~~~
rtpdump.c:733:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 14].enc = "MPA "; pt_map[ 14].rate = 90000; pt_map[ 14].ch = 0;
                  ^ ~~~~~~
rtpdump.c:734:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 15].enc = "G728"; pt_map[ 15].rate =  8000; pt_map[ 15].ch = 1;
                  ^ ~~~~~~
rtpdump.c:735:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 16].enc = "DVI4"; pt_map[ 16].rate = 11025; pt_map[ 16].ch = 1;
                  ^ ~~~~~~
rtpdump.c:736:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 17].enc = "DVI4"; pt_map[ 17].rate = 22050; pt_map[ 17].ch = 1;
                  ^ ~~~~~~
rtpdump.c:737:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 18].enc = "G729"; pt_map[ 18].rate =  8000; pt_map[ 18].ch = 1;
                  ^ ~~~~~~
rtpdump.c:738:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 23].enc = "SCR "; pt_map[ 23].rate = 90000; pt_map[ 23].ch = 0; /*r*/
                  ^ ~~~~~~
rtpdump.c:739:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 24].enc = "MPEG"; pt_map[ 24].rate = 90000; pt_map[ 24].ch = 0; /*r*/
                  ^ ~~~~~~
rtpdump.c:740:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 25].enc = "CelB"; pt_map[ 25].rate = 90000; pt_map[ 25].ch = 0;
                  ^ ~~~~~~
rtpdump.c:741:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 26].enc = "JPEG"; pt_map[ 26].rate = 90000; pt_map[ 26].ch = 0;
                  ^ ~~~~~~
rtpdump.c:742:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 27].enc = "CUSM"; pt_map[ 27].rate = 90000; pt_map[ 27].ch = 0; /*r*/
                  ^ ~~~~~~
rtpdump.c:743:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 28].enc = "nv  "; pt_map[ 28].rate = 90000; pt_map[ 28].ch = 0;
                  ^ ~~~~~~
rtpdump.c:744:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 29].enc = "PicW"; pt_map[ 29].rate = 90000; pt_map[ 29].ch = 0; /*r*/
                  ^ ~~~~~~
rtpdump.c:745:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 30].enc = "CPV "; pt_map[ 30].rate = 90000; pt_map[ 30].ch = 0; /*r*/
                  ^ ~~~~~~
rtpdump.c:746:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 31].enc = "H261"; pt_map[ 31].rate = 90000; pt_map[ 31].ch = 0;
                  ^ ~~~~~~
rtpdump.c:747:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 32].enc = "MPV "; pt_map[ 32].rate = 90000; pt_map[ 32].ch = 0;
                  ^ ~~~~~~
rtpdump.c:748:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 33].enc = "MP2T"; pt_map[ 33].rate = 90000; pt_map[ 33].ch = 0;
                  ^ ~~~~~~
rtpdump.c:749:19: warning: assigning to 'char *' from 'const char [5]' discards
      qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
  pt_map[ 34].enc = "H263"; pt_map[ 34].rate = 90000; pt_map[ 34].ch = 0;
                  ^ ~~~~~~
60 warnings generated.
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -o rtpdump hpt.o host2ip.o                     rd.o rtpdump.o 
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c notify.c
notify.c:67:6: warning: no previous prototype for function 'check_clr_fd'
      [-Wmissing-prototypes]
void check_clr_fd(void)
     ^
notify.c:137:13: warning: no previous prototype for function
      'notify_set_itimer_func' [-Wmissing-prototypes]
Notify_func notify_set_itimer_func(
            ^
notify.c:204:63: warning: format specifies type 'unsigned long' but the argument
      has type '__darwin_suseconds_t' (aka 'int') [-Wformat]
      fprintf(stderr, "Timeout: %lu.%06lu\n", timeout.tv_sec, timeout.tv_usec);
                                    ~~~~~                     ^~~~~~~~~~~~~~~
                                    %06d
3 warnings generated.
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c multimer.c
multimer.c:91:6: warning: no previous prototype for function 'timer_check'
      [-Wmissing-prototypes]
void timer_check(void)
     ^
multimer.c:115:17: warning: no previous prototype for function 'timer_set'
      [-Wmissing-prototypes]
struct timeval *timer_set(struct timeval *interval, 
                ^
multimer.c:187:17: warning: no previous prototype for function 'timer_get'
      [-Wmissing-prototypes]
struct timeval *timer_get(struct timeval *timeout)
                ^
multimer.c:227:5: warning: no previous prototype for function 'timer_pending'
      [-Wmissing-prototypes]
int timer_pending(void)
    ^
4 warnings generated.
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c rtpplay.c
rtpplay.c:354:52: warning: passing 'char *' to parameter of type
      'unsigned char *' converts between pointers to integer types with
      different sign [-Wpointer-sign]
    if (hpt(argv[optind], (struct sockaddr *)&sin, &ttl) < 0) {
                                                   ^~~~
rtpplay.c:310:63: note: passing argument to parameter 'ttl' here
  extern int hpt(char *h, struct sockaddr *sa, unsigned char *ttl);
                                                              ^
rtpplay.c:358:29: warning: comparison of integers of different signs:
      'in_addr_t' (aka 'unsigned int') and 'int' [-Wsign-compare]
    if (sin.sin_addr.s_addr == -1) {
        ~~~~~~~~~~~~~~~~~~~ ^  ~~
2 warnings generated.
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -o rtpplay hpt.o host2ip.o notify.o multimer.o rd.o rtpplay.o 
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c rtpsend.c
rtpsend.c:240:6: warning: initializing 'char *' with an expression of type
      'const char [4]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"end",   RTCP_SDES_END},
     ^~~~~
rtpsend.c:241:6: warning: initializing 'char *' with an expression of type
      'const char [6]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"cname", RTCP_SDES_CNAME},
     ^~~~~~~
rtpsend.c:242:6: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"name",  RTCP_SDES_NAME},
     ^~~~~~
rtpsend.c:243:6: warning: initializing 'char *' with an expression of type
      'const char [6]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"email", RTCP_SDES_EMAIL},
     ^~~~~~~
rtpsend.c:244:6: warning: initializing 'char *' with an expression of type
      'const char [6]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"phone", RTCP_SDES_PHONE},
     ^~~~~~~
rtpsend.c:245:6: warning: initializing 'char *' with an expression of type
      'const char [4]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"loc",   RTCP_SDES_LOC},
     ^~~~~
rtpsend.c:246:6: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"tool",  RTCP_SDES_TOOL},
     ^~~~~~
rtpsend.c:247:6: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"note",  RTCP_SDES_NOTE}, 
     ^~~~~~
rtpsend.c:248:6: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    {"priv",  RTCP_SDES_PRIV}, 
     ^~~~~~
rtpsend.c:401:7: warning: no previous prototype for function 'usec2ntp'
      [-Wmissing-prototypes]
u_int usec2ntp(u_int usec)
      ^
rtpsend.c:599:7: warning: initializing 'char *' with an expression of type
      'const char [5]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    { "SDES",  rtcp_write_sdes },
      ^~~~~~
rtpsend.c:600:7: warning: initializing 'char *' with an expression of type
      'const char [3]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    { "RR",  rtcp_write_rr },
      ^~~~
rtpsend.c:601:7: warning: initializing 'char *' with an expression of type
      'const char [3]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    { "SR",  rtcp_write_sr },
      ^~~~
rtpsend.c:602:7: warning: initializing 'char *' with an expression of type
      'const char [4]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    { "BYE",  rtcp_write_bye },
      ^~~~~
rtpsend.c:603:7: warning: initializing 'char *' with an expression of type
      'const char [4]' discards qualifiers
      [-Wincompatible-pointer-types-discards-qualifiers]
    { "APP",  rtcp_write_app },
      ^~~~~
rtpsend.c:802:24: warning: missing field 'data' initializer
      [-Wmissing-field-initializers]
  } packet = { 0, -1, 0};
                       ^
rtpsend.c:918:52: warning: passing 'char *' to parameter of type
      'unsigned char *' converts between pointers to integer types with
      different sign [-Wpointer-sign]
    if (hpt(argv[optind], (struct sockaddr *)&sin, &ttl) < 0) {
                                                   ^~~~
rtpsend.c:877:63: note: passing argument to parameter 'ttl' here
  extern int hpt(char *h, struct sockaddr *sa, unsigned char *ttl);
                                                              ^
rtpsend.c:922:29: warning: comparison of integers of different signs:
      'in_addr_t' (aka 'unsigned int') and 'int' [-Wsign-compare]
    if (sin.sin_addr.s_addr == -1) {
        ~~~~~~~~~~~~~~~~~~~ ^  ~~
18 warnings generated.
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -o rtpsend hpt.o host2ip.o notify.o multimer.o      rtpsend.o 
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -c rtptrans.c
rtptrans.c:93:5: warning: no previous prototype for function 'create_stream'
      [-Wmissing-prototypes]
int create_stream(int addr, int next)
    ^
rtptrans.c:149:5: warning: no previous prototype for function 'find_stream'
      [-Wmissing-prototypes]
int find_stream(int addr, int ts, int next, int m)
    ^
rtptrans.c:318:49: warning: comparison of integers of different signs: 'ssize_t'
      (aka 'long') and 'unsigned long' [-Wsign-compare]
          if ((sendmsg(side[i][2].sock, &msg,0))!= 
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
rtptrans.c:388:6: warning: no previous prototype for function 'usage'
      [-Wmissing-prototypes]
void usage(char *argv0)
     ^
rtptrans.c:461:37: warning: comparison of integers of different signs:
      'in_addr_t' (aka 'unsigned int') and 'int' [-Wsign-compare]
    if (host[i].sin.sin_addr.s_addr == -1) {
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^  ~~
5 warnings generated.
cc -g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter -o rtptrans hpt.o host2ip.o notify.o multimer.o      rtptrans.o 
~/Projects/rtptools(master) $ echo $?
0
```

--- a/Makefile
+++ b/Makefile
@@ -73,26 +73,13 @@ COMPAT_OBJS = \
 OBJS =	$(rtpdump_OBJS) $(rtpplay_OBJS) $(rtpsend_OBJS) $(rtptrans_OBJS)
 OBJS +=	$(COMPAT_OBJS)
 
-# Allegedly, windows need these _empty_ include files.
-# Please say that isn't so and get rid of this.
-EMPTY = \
-	win/include/arpa/inet.h		\
-	win/include/netdb.h		\
-	win/include/netinet/in.h	\
-	win/include/sys/select.h	\
-	win/include/sys/socket.h	\
-	win/include/sys/time.h		\
-	win/include/sys/uio.h		\
-	win/include/unistd.h
-
 WINDOWS = \
 	win/rtptools.sln				\
 	win/rtpdump.vcxproj win/rtpplay.vcxproj		\
 	win/rtpsend.vcxproj win/rtptrans.vcxproj	\
 	win/gettimeofday.c win/gettimeofday.h		\
 	win/winsocklib.c win/winsocklib.h		\
-	win/getopt.c win/getopt.h			\
-	$(EMPTY)
+	win/getopt.c win/getopt.h
 
 DISTFILES = \
 	LICENSE			\
@@ -152,10 +139,6 @@ rtpsend: $(rtpsend_OBJS)
 
 rtptrans: $(rtptrans_OBJS)
 	$(CC) $(CFLAGS) -o rtptrans $(rtptrans_OBJS) $(LDADD)
-
-$(EMPTY):
-	mkdir -p win/include/{arpa,netinet,sys}
-	touch $@
 
 # --- maintainer targets ---
 

--- a/Makefile
+++ b/Makefile
@@ -113,8 +113,8 @@ distclean: clean
 
 clean:
 	rm -f $(TARBALL) $(BINS) $(OBJS) $(HTML)
-	rm -rf rtptools-$(VERSION) win/include
-	rm -rf *.dSYM *.core *~ .*~
+	rm -rf *.dSYM *.core *~ .*~ win/*~
+	rm -rf rtptools-$(VERSION)
 
 install: $(PROG) $(MAN1)
 	install -d $(BINDIR)      && install -m 0755 $(PROG) $(BINDIR)

--- a/host2ip.c
+++ b/host2ip.c
@@ -29,11 +29,15 @@
  */
 
 #include <sys/types.h>
-#include <sys/socket.h>      /* struct sockaddr */
-#include <netdb.h>           /* gethostbyname() */
-#include <netinet/in.h>      /* sockaddr_in */
-#include <arpa/inet.h>       /* inet_addr() */
-#include <string.h>          /* strlen() added by Akira 12/27/01 */ 
+#include <string.h>
+
+#ifndef WIN32
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <netdb.h>
+#endif
+
 #include "sysdep.h"
 
 /*

--- a/hpt.c
+++ b/hpt.c
@@ -28,19 +28,22 @@
  * SUCH DAMAGE.
  */
 
-/*
-* Parse [host]/port[/ttl].  Return 0 if ok, -1 if error; set sockaddr and
-* ttl value.
-*/
+
+#include <sys/types.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/types.h>
+
+#ifndef WIN32
 #include <sys/socket.h>
 #include <netinet/in.h>
+#endif
+
 #include "sysdep.h"
 
 extern struct in_addr host2ip(char *);
 
+/* Parse [host]/port[/ttl]. Return 0 if ok, -1 on error;
+ * set sockaddr and ttl value. */
 int hpt(char *h, struct sockaddr *sa, unsigned char *ttl)
 {
   char *s;

--- a/multimer.c
+++ b/multimer.c
@@ -40,13 +40,17 @@
 /*                                                                          */
 /****************************************************************************/
 
-#include "notify.h"      /* Notify_func */
-#include <stdio.h>
-#include <stdlib.h>      /* */
 #include <sys/types.h>
-#include <sys/time.h>    /* timeval, gettimeofday() */
 #include <assert.h>
-#include "sysdep.h"      /* system-dependent */
+#include <stdlib.h>
+#include <stdio.h>
+
+#ifndef WIN32
+#include <sys/time.h>
+#endif
+
+#include "sysdep.h"
+#include "notify.h"
 
 typedef struct TQE {
     struct TQE *link;           /* link to next timer */

--- a/notify.c
+++ b/notify.c
@@ -5,13 +5,17 @@ notify  --  primitive notification service implementing a subset of the SunOS
 Copyright 1993 by AT&T Bell Laboratories; all rights reserved
 */
 
-#include <sys/time.h>
+#include <stdlib.h>
+#include <string.h>
 #include <signal.h>
-#include <stdio.h>     /*DEBUG, fprintf() */
-#include <stdlib.h>    /* malloc() */
-#include <unistd.h>    /* select() */
-#include <errno.h>     /* EINTR,    Added by Akira 12/11/01 */
-#include <string.h>    /* memset(), Added by Akira 12/11/01 */
+#include <stdio.h>
+#include <errno.h>
+
+#ifndef WIN32
+#include <sys/select.h>
+#include <sys/time.h>
+#endif
+
 #include "sysdep.h"
 #include "notify.h"
 #include "multimer.h"

--- a/notify.h
+++ b/notify.h
@@ -5,10 +5,6 @@ stripped-down version of <xview/notify.h>
 patents pending in the U.S. and foreign countries. OPEN LOOK is
 a trademark of USL. Used by written permission of the owners.
 */
-#ifndef _notify_h
-#define _notify_h
-#include <sys/types.h>
-#include <sys/time.h>
 
 /*
  * Client notification function return values for notifier to client calls.
@@ -114,5 +110,3 @@ extern Notify_func_signal notify_set_signal_func(Notify_client nclient,
 * Initialize Readfds (flag = 0), Writefds (1), Exceptfds (2).
 */
 extern void notify_set_socket(int sock, int flag);
-
-#endif

--- a/rd.c
+++ b/rd.c
@@ -28,15 +28,20 @@
  * SUCH DAMAGE.
  */
 
-#include <stdio.h>
 #include <sys/types.h>
-#include <sys/socket.h>  /* struct sockaddr */
-#include <netinet/in.h>
-#include <arpa/inet.h>   /* inet_ntoa() */
-#include <string.h>      /* strncmp() */
+#include <string.h>
+#include <stdio.h>
+#include <time.h>
+
+#ifndef WIN32
 #include <sys/time.h>
-#include <time.h>        /* localtime() added by Akira 12/27/01 */
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#endif
+
 #include "rtpdump.h"
+
 #define RTPFILE_VERSION "1.0"
 
 /*

--- a/rtpdump.c
+++ b/rtpdump.c
@@ -29,22 +29,30 @@
  */
 
 #include <sys/types.h>
+#include <stdlib.h>
+
+#ifdef WIN32
+#include <fcntl.h>       /* O_BINARY */
+#else
+#include <unistd.h>
+#include <sys/time.h>
 #include <sys/socket.h>
-#include <netinet/in.h>  /* struct sockaddr_in */
-#include <arpa/inet.h>   /* inet_ntoa() */
-#include <sys/time.h>    /* gettimeofday() */
+#include <sys/select.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#endif
+
+#include <signal.h>
 #include <string.h>
 #include <stdio.h>
-#include <signal.h>
 #include <errno.h>
-#include <unistd.h>      /* select(), perror() */
-#include <stdlib.h>      /* getopt(), atoi() */
-#include <fcntl.h>       /* O_BINARY on Win32 */
+
 #include "rtp.h"
 #include "vat.h"
 #include "rtpdump.h"
 #include "ansi.h"
 #include "sysdep.h"
+
 #define RTPFILE_VERSION "1.0"
 
 typedef uint32_t member_t;

--- a/rtpplay.c
+++ b/rtpplay.c
@@ -28,28 +28,29 @@
  * SUCH DAMAGE.
  */
 
-#include <stdint.h>
+
 #include <sys/types.h>
-#include <sys/time.h>    /* gettimeofday() */
-#include <sys/socket.h>  /* struct sockaddr */
-#include <netinet/in.h>
-#include <arpa/inet.h>   /* inet_ntoa() */
-#include <netdb.h>       /* gethostbyname() */
-#include <time.h>
-#include <stdio.h>       /* stderr, printf() */
+#include <search.h>
+#include <stdint.h>
+#include <stdlib.h>
 #include <string.h>
-#include <stdlib.h>      /* perror() */
-#include <unistd.h>      /* write() */
-#include "sysdep.h"
-#if HAVE_SEARCH_H
-#include <search.h>      /* hash table */
-#else
-#include "hsearch.h"
+#include <stdio.h>
+#include <time.h>
+
+#ifndef WIN32
+#include <unistd.h>
+#include <sys/time.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <netdb.h>
 #endif
-#include "notify.h"      /* notify_start(), ... */
-#include "rtp.h"         /* RTP headers */
-#include "rtpdump.h"     /* RD_packet_t */
-#include "multimer.h"    /* timer_set() */
+
+#include "sysdep.h"
+#include "notify.h"
+#include "rtp.h"
+#include "rtpdump.h"
+#include "multimer.h"
 #include "ansi.h"
 
 #define READAHEAD 16 /* must be power of 2 */

--- a/rtpsend.c
+++ b/rtpsend.c
@@ -28,22 +28,25 @@
  * SUCH DAMAGE.
  */
 
+#include <sys/types.h>
+#include <stdlib.h>
+#include <string.h>
 #include <stdio.h>
 #include <ctype.h>
-#include <sys/types.h>
-#include <sys/time.h>    /* gettimeofday() */
-#include <sys/socket.h>  /* struct sockaddr */
-#include <netinet/in.h>
-#include <arpa/inet.h>   /* inet_ntoa() */
-#include <netdb.h>       /* gethostbyname() */
 #include <time.h>
-#include <stdio.h>       /* stderr, printf() */
-#include <string.h>
-#include <stdlib.h>      /* perror(), calloc() */
-#include <unistd.h>      /* write() */
-#include "notify.h"      /* notify_start(), ... */
-#include "rtp.h"         /* RTP headers */
-#include "multimer.h"    /* timer_set() */
+
+#ifndef WIN32
+#include <unistd.h>
+#include <sys/time.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <netdb.h>
+#endif
+
+#include "notify.h"
+#include "rtp.h"
+#include "multimer.h"
 #include "ansi.h"
 #include "sysdep.h"
 

--- a/rtptrans.c
+++ b/rtptrans.c
@@ -29,18 +29,19 @@
  */
 
 #include <sys/types.h>
-#include <sys/uio.h>
-#include <sys/socket.h>
-#include <netinet/in.h>  /* struct sockaddr_in */
-#include <sys/time.h>    /* gettimeofday() */
-#include <arpa/inet.h>   /* inet_ntoa() */
+#include <signal.h>
+#include <stdlib.h>
 #include <string.h>
 #include <stdio.h>
-#include <signal.h>
 #include <errno.h>
-#include <unistd.h>      /* select(), perror() */
-#include <stdlib.h>      /* getopt(), atoi() */
-#include <memory.h>      /* memset() */
+
+#ifndef WIN32
+#include <unistd.h>
+#include <sys/time.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#endif
 
 #include "config.h"
 #include "rtp.h"

--- a/win/rtpdump.vcxproj
+++ b/win/rtpdump.vcxproj
@@ -105,7 +105,6 @@
     <ClInclude Include="include\sys\select.h" />
     <ClInclude Include="include\sys\socket.h" />
     <ClInclude Include="include\sys\time.h" />
-    <ClInclude Include="include\sys\uio.h" />
     <ClInclude Include="include\unistd.h" />
     <ClInclude Include="winsocklib.h" />
   </ItemGroup>

--- a/win/rtpplay.vcxproj
+++ b/win/rtpplay.vcxproj
@@ -108,7 +108,6 @@
     <ClInclude Include="include\sys\select.h" />
     <ClInclude Include="include\sys\socket.h" />
     <ClInclude Include="include\sys\time.h" />
-    <ClInclude Include="include\sys\uio.h" />
     <ClInclude Include="include\unistd.h" />
     <ClInclude Include="winsocklib.h" />
   </ItemGroup>

--- a/win/rtpsend.vcxproj
+++ b/win/rtpsend.vcxproj
@@ -107,7 +107,6 @@
     <ClInclude Include="include\sys\select.h" />
     <ClInclude Include="include\sys\socket.h" />
     <ClInclude Include="include\sys\time.h" />
-    <ClInclude Include="include\sys\uio.h" />
     <ClInclude Include="include\unistd.h" />
     <ClInclude Include="winsocklib.h" />
   </ItemGroup>

--- a/win/rtptrans.vcxproj
+++ b/win/rtptrans.vcxproj
@@ -105,7 +105,6 @@
     <ClInclude Include="include\sys\select.h" />
     <ClInclude Include="include\sys\socket.h" />
     <ClInclude Include="include\sys\time.h" />
-    <ClInclude Include="include\sys\uio.h" />
     <ClInclude Include="include\unistd.h" />
     <ClInclude Include="winsocklib.h" />
   </ItemGroup>

